app/src/main/java/org/fossasia/openevent/general/AuthActivity.kt

```diff
@@ -68,4 +74,13 @@ class AuthActivity : AppCompatActivity() {
         super.onBackPressed()
         startActivity(Intent(this, MainActivity::class.java).setFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP))
     }
+
+    private fun isSnackbar() {
```

Comment:
isSnackbar should return a boolean and tell if this activity is a snackbar with no side effects